cerberus/cerberus.py

```diff
@@ -548,7 +548,7 @@ def _validate_dependencies(self, definition, field, value):
     def __validate_dependencies_mapping(self, dependencies, field):
         validated_deps = 0
         for dep_name, dep_values in dependencies.items():
-            if isinstance(dep_values, _str_type):
+            if isinstance(dep_values, (_str_type, bool)):
```

Comment:
please test more generally:
`if not isinstance(dep_values, Sequence) or isinstance (dep_values, _str_type)`